src/components/Navbar.tsx

```diff
@@ -1,23 +1,99 @@
 import React from "react";
-import tw from "twin.macro";
+import tw, { styled } from "twin.macro";
 
-import Button from "./Button";
-import { useConnection } from "../hooks";
+import { ReactComponent as Logo } from "../assets/logo.svg";
+import { Twitter, Medium, Github, Discord, Discourse } from "../assets/icons";
+import Link from "./Link";
+import { MaxWidthWrapper } from "./Wrappers";
 
-const Navbar: React.FC = () => {
-  const { connect, address, signer, network } = useConnection();
-  return (
-    <Container>
-      <code>
-        <pre>{JSON.stringify({ address, signer, network }, null, 2)}</pre>
-      </code>
-      <ConnectButton onClick={connect}>Connect</ConnectButton>
-    </Container>
-  );
-};
+const links = [
+  {
+    name: "How UMA Works",
+    to: "https://github.com/",
+  },
+  {
+    name: "Getting Started",
+    to: "https://github.com",
+  },
+  {
+    name: "Docs",
+    to: "https://github.com",
+  },
+  {
+    name: "Vote",
+    to: "https://github.com",
+  },
+  { name: "KPI Options", to: "/" },
+];
+const Navbar: React.FC = () => (
+  <Wrapper>
+    <NavWrapper as="nav">
+      <Logo />
+      <LinksWrapper>
+        <LinksList>
+          {links.map((link) => (
+            <li key={link.name}>
+              <NavLink href={link.to}>{link.name}</NavLink>
+            </li>
+          ))}
+        </LinksList>
+        <IconsList>
+          <IconItem>
+            <Link href="" rel="noopener norefferrer" target="_blank">
+              <Twitter />
+            </Link>
+          </IconItem>
+          <IconItem>
+            <Link href="" rel="noopener norefferrer" target="_blank">
+              <Discord />
+            </Link>
+          </IconItem>
+          <IconItem>
+            <Link href="" rel="noopener norefferrer" target="_blank">
+              <Discourse />
+            </Link>
+          </IconItem>
+          <IconItem>
+            <Link href="" rel="noopener norefferrer" target="_blank">
+              <Medium />
+            </Link>
+          </IconItem>
+          <IconItem>
+            <Link href="" rel="noopener norefferrer" target="_blank">
+              <Github />
+            </Link>
+          </IconItem>
+        </IconsList>
+      </LinksWrapper>
+    </NavWrapper>
+  </Wrapper>
+);
 
-const Container = tw.div`flex justify-between w-full border-b p-4 shadow-md`;
-const ConnectButton = tw(Button)`
-    p-2 bg-red-500 
-`;
 export default Navbar;
+
+const Wrapper = tw.div`w-full bg-secondary`;
+const NavWrapper = tw(MaxWidthWrapper)`
+    flex justify-between items-center px-10 pb-14 pt-12
+    
+`;
+const NavLink = tw(Link)`
+      font-semibold hocus:text-primary transition-colors duration-300
+`;
+const LinksWrapper = tw.div`flex`;
+const List = tw.ul`flex`;
+const IconsList = styled(List)`
+  margin-left: 50px;
+  > * + * {
```

Comment:
what does this do?

Reply:
The selector `> * + *` is a combination of CSS selectors and does this:
It takes any element that is a direct descendant of an `IconsList` (this is done with `>` selector) that is followed by any other element (this is the `* + *` selector) and styles it as defined below.